Documentation/gettingstarted/hubble.rst

```diff
@@ -10,43 +10,109 @@
 Networking and security observability with Hubble
 *************************************************
 
-This guide provides a walkthrough of setting up a local multi-node Kubernetes
-cluster on Docker using `kind <https://kind.sigs.k8s.io/>`_ in order to
-demonstrate some of Hubble's capabilities.
+This guide provides a walkthrough of setting up a local Kubernetes cluster with
+Hubble and Cilium installed, in order to demonstrate some of Hubble's
+capabilities.
 
 If you haven't read the :ref:`intro` yet, we'd encourage you to do that first.
 
 The best way to get help if you get stuck is to ask a question on the `Cilium
 Slack channel <https://cilium.herokuapp.com>`_.  With Cilium contributors
 across the globe, there is almost always someone available to help.
 
-.. include:: kind-install.rst
+Set up a Kubernetes cluster
+===========================
+
+To run a Kubernetes cluster on your local machine, you have the choice to
+either set up a single-node cluster with
+`minikube <https://kubernetes.io/docs/setup/learning-environment/minikube/>`_,
+or a local multi-node cluster on Docker using
+`kind <https://kind.sigs.k8s.io/>`_:
+
+- ``minikube`` runs a single-node Kubernetes cluster inside a Virtual Machine
+  (VM) and is the easiest way to run a Kubernetes cluster locally.
+- ``kind`` runs a multi-node Kubernetes using Docker container to emulate
+  cluster nodes. It allows you to experiment with the cluster-wide observability
+  features of Hubble Relay.
+
+When unsure about the option to pick, follow the instructions for ``minikube``
+as it is less likely to cause friction.
+
+.. tabs::
+
+    .. group-tab:: Single-node cluster with ``minikube``
+
+        **Install kubectl & minikube**
+
+        .. include:: minikube-install.rst
```

Comment:
not directly related to this PR, but do we still need to run:
```
minikube ssh -- sudo mount bpffs -t bpf /sys/fs/bpf
```
i don't remember running this command for a while :)

Reply:
Mounting the BPF filesystem is not required for testing, but it is highly recommended, as we require it for production use. I don't think nodeinit will mount it for us, but I might be wrong.

But yes, this tutorial should work without it, but I guess it's best practice to recommend it anyway.